class.application-passwords.php

```diff
@@ -228,6 +244,56 @@ public static function create_new_application_password( $user_id, $name ) {
 	}
 
 	/**
+	 * Create application password via ajax
+	 *
+	 * @since 0.1-dev
+	 *
+	 * @access public
+	 * @static
+	 */
+	public static function ajax_create_application_password() {
+		$user_id = get_current_user_id();
+		$name = $_POST['app_name'];
+		if ( ! isset( $_POST['create_application_password'] ) || wp_verify_nonce( $_POST['_nonce_create_application_password'], 'create_application_password') ) {
```

Comment:
Why would we allow this to proceed if `$_POST['create_application_password']` is not set?